documentation/book/getting-started.adoc

```diff
@@ -9,7 +9,9 @@
 [id='getting-started-{context}']
 = Getting started with {ProductName}
 
-{ProductName} works on all types of clusters, from public and private clouds to local deployments intended for development.
+{ProductName} works on all types of Kubernetes cluster irrespective of distribution, from public and private clouds to local deployments intended for development.
+{ProductName} supports a few features which are specific to OpenShift, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.
```

Comment:
Should we now call it OKD in upstream instead of OpenShift? Or do we think OpenShift is ok here?

Reply:
I wonder how many people know what OKD is?

Reply:
Yeah, the same feeling we don't say OKD either:-)

Reply:
```suggestion
{ProductName} supports a few features which are specific to OpenShift/OKD, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.
```

?

Reply:
I suggest "specific to OpenShift Container Platform and OKD".

Reply:
```suggestion
{ProductName} supports a few features which are specific to OpenShift and OKD, where such integration benefits OpenShift users and cannot be implemented equivalently using standard Kubernetes.
```

Reply:
I don't think it makes sense to use both OpenShift and OKD. The point is the OpenShift is a name of commercial offering. OKD isn't.

Reply:
Mentioning OKD here, when other parts of the doc talk about OpenShift is inconsistent. We're we're going to change to OKD here, then why not everywhere?

Reply:
That was of course a generic comment applying to all usages. And I agree that nobody knows what OKD is. I think saying _OpenShift and OKD_ is pointless.

I do not mind if OpenShift is used here. I just thought that if people complain that the docs are too OpenShift specific it is for sure also because it is a name of a commercial product. I thought one could mitigate that by using OKD. But if you think its not needed, lets keep using OPenShift

Reply:
I'm not opposed to changing it to OKD, but until someone objects specifically to the remaining OpenShift occurrences on the basis of it being commercial (so that changing it to OKD would remedy the situation) then I think it's just more helpful for the reader to leave it as OpenShift (since they're more likely to already know what that is).